lib/ruby_aem_aws/constants.rb

```diff
@@ -42,7 +42,7 @@ class ELBInstanceState
   end
 
   class Constants
-    REGION_DEFAULT = 'ap-southeast-2'.freeze
+    REGION_DEFAULT = ENV['AWS_DEFAULT_REGION'] || ENV['aws_default_region'] || 'ap-southeast-2'.freeze
```

Comment:
Hi, I know `'AWS_DEFAULT_REGION'` is the correct name, but maybe the deprecated `'AWS_REGION'` could be the second fallback?
It is a bit overkill, but it could potentislly save some poor souls who still use the old name?
What do you think?

For posterity - I mean:
`REGION_DEFAULT = ENV['AWS_DEFAULT_REGION'] || ENV['aws_default_region'] || ENV['AWS_REGION'] || ENV['aws_region'] ||'ap-southeast-2'.freeze`

Reply:
I think this shouldn't be a big deal to support the deprecated env variable as well.